src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -232,6 +233,31 @@ Napi::Value CoreWrap::get_available_devices(const Napi::CallbackInfo& info) {
     return js_devices;
 }
 
+Napi::Value CoreWrap::get_versions(const Napi::CallbackInfo& info) {
+    if (info.Length() == 0) {
+        reportError(info.Env(), "No argument provided in the getVersions() method call.");
```

Comment:
Include useful info into error message
```suggestion
        reportError(info.Env(), "get_versions expects 1 argument of string type");
```